### PR TITLE
Add resource call page logic containing an anonymous call

### PR DESCRIPTION
**Pull request by Archifer**

### CODE REVIEW

File: src/components/Navigation-bar.vue [hunk 1]
```diff
@@ -19,9 +19,13 @@
...
       redirect: function () {
-        if ('PendingPushRequests' === this.$route.name) {
-          this.$router.back();
-        }
+        onegini.user.getAuthenticatedUserProfile()
+        .then((_) => {
+          this.$router.push('dashboard')
+        })
+        .catch((_) => {
+          this.$router.push('login')
```
Comment (MichielVrins) [inline src/components/Navigation-bar.vue:27]: end with ;

File: src/components/Navigation-bar.vue [hunk 2]
```diff
@@ -19,9 +19,13 @@
   export default {
     methods: {
       redirect: function () {
-        if ('PendingPushRequests' === this.$route.name) {
-          this.$router.back();
-        }
+        onegini.user.getAuthenticatedUserProfile()
+        .then((_) => {
```
Comment (MichielVrins) [inline src/components/Navigation-bar.vue:23]: I think you can ommit the _

File: src/views/App.vue [hunk 3]
```diff
@@ -66,6 +66,7 @@
             return onegini.user.getAuthenticatedUserProfile();
           })
           .catch((err) => {
+            console.log(err);
```
Comment (MichielVrins) [inline src/views/App.vue:69]: probably don't need this, do we?

File: src/components/Navigation-bar.vue (hunk 2 above)
Comment (Archifer) [reply src/components/Navigation-bar.vue:23]: think this following the conventions related to unused params more

File: src/views/App.vue (hunk 3 above)
Comment (Archifer) [reply src/views/App.vue:69]: this is usefull for debugging in the future; as the err.description is more generic or when it is forced to the login page; always having the full error object in the logs is nice to have as a dev

**Review by MichielVrins — APPROVED** [review]

**Pull request merged**